### DISCUSSION TITLE
Starstruck Exchange

### DISCUSSION THREAD

**Discussion by mamadaliev** [Schweinepriester/github-profile-achievements#706] [General]
Hello! 
Star me and I'll star you back.

https://github.com/mamadaliev/spring-boot-postgres-vue-docker

**Comment by PARKER-X** [comment]
https://github.com/PARKER-X/BarBar

**Reply by mamadaliev** [reply]
@PARKER-X Done. Nice to do business with you!

**Comment by 4331000773** [comment]
167934

**Comment by elijahpereira** [comment]
star for star https://elijahpereira.github.io/

**Reply by mamadaliev** [reply]
@elijahpereira Done. Thank you!

**Reply by ArielMAJ** [reply]
@elijahpereira I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by nguyen-tuan-hiep** [reply]
Done. Can you give me a star on: https://github.com/nguyen-tuan-hiep/Introduction-to-Communication-Engineering

**Comment by 0x4248** [comment]
Done [can you star mine](https://www.github.com/awesomelewis2007/lab_space).

**Reply by ArielMAJ** [reply]
@awesomelewis2007 I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by 0x4248** [reply]
Done.

**Reply by ArielMAJ** [reply]
Ty!

**Reply by mamadaliev** [reply]
@awesomelewis2007 Done too.

**Reply by vinograd0v** [reply]
> Done [can you star mine](https://www.github.com/awesomelewis2007/lab_space).

Done https://github.com/mmanosalva/SublimeTeX

**Comment by ArielMAJ** [comment]
> Hello! Star me and I'll star you back.
> 
> https://github.com/mamadaliev/spring-boot-postgres-vue-docker

@mamadaliev I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by mamadaliev** [reply]
Done. Thank you!

**Comment by Veercodeprog** [comment]
You can take part in ongoing [coding problem](https://github.com/Veercodeprog/Data-Structures-implementation/discussions/categories/coding-discussion-share-your-solution-with-best-time-complexity-approach) discussion.

A daily problem will be posted, and the answer with the most efficient time complexity submitted first will be selected as the accepted solution.

**Reply by ArielMAJ** [reply]
Ok

**Comment by Veercodeprog** [comment]
https://github.com/Veercodeprog/full-stack-assignment. done

**Reply by mamadaliev** [reply]
Thank you! Done.

**Comment by anamanuellar** [comment]
Hello!
Star me and I'll star you back.
https://github.com/anamanuellar/anamanuellar

**Reply by ArielMAJ** [reply]
@anamanuellar I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by anamanuellar** [reply]
Thanks. I starred you too.

**Reply by ArielMAJ** [reply]
Tyvm!

**Reply by mamadaliev** [reply]
@anamanuellar Thank you! I starred you too.

**Reply by anamanuellar** [reply]
Thanks @mamadaliev

**Reply by elijahpereira** [reply]
Done please star https://github.com/elijahpereira/elijahpereira.github.io

**Reply by elijahpereira** [reply]
> Hello!
> 
> Star me and I'll star you back.
> 
> https://github.com/anamanuellar/anamanuellar 

Done star https://github.com/elijahpereira/elijahpereira.github.io

**Comment by vinograd0v** [comment]


**Reply by mamadaliev** [reply]
@mmanosalva Thank you! Done too.

**Comment by xommmax** [comment]
HI everyone!
[Flutter Guidelines](https://github.com/4lfant/flutter-guidelines) repository contains my personal recommendations for policies, procedures, and standards regarding mobile development. I collected all my knowledge in this document.
Please star it 🥰

**Reply by mamadaliev** [reply]
Done.

**Comment by nguyen-tuan-hiep** [comment]
Can you guys give a star for my repo: https://github.com/nguyen-tuan-hiep/Introduction-to-Communication-Engineering. I will star you back

**Reply by ArielMAJ** [reply]
Done

**Comment by DhrubajyotiBiswasfounderofDribll** [comment]
I have starred your repo. Can you star me back and collaborate: https://github.com/DSoftofficial/Tilder

> Hello! Star me and I'll star you back.
> 
> https://github.com/mamadaliev/spring-boot-postgres-vue-docker

**Reply by ArielMAJ** [reply]
@dhrubajyotifounderofDSoft I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by 1wos** [comment]
I marked your repo with stars! Can you do it for me, too?

https://github.com/sommizzu/Big-Data-Convergence

**Reply by ArielMAJ** [reply]
@sommizzu I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by mamadaliev** [reply]
@sommizzu Done.

**Comment by MrAzimzadeh** [comment]
Star this repository and I will star your repository
https://github.com/MrAzimzadeh/EcommerceK101

**Reply by ArielMAJ** [reply]
@MrAzimzadeh I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by mamadaliev** [reply]
@MrAzimzadeh Done. Thank you.

**Comment by YuZhangWang** [comment]
done
[YuZhangWang](https://github.com/YuZhangWang/YuZhangWang)
